config.lua

```diff
@@ -7,6 +7,7 @@ config({'development'}, {
     password = "pleasechangeme"
   },
   secret = "pleasechangeme",
+  session_name = "mudlet-package-repo-session"
```

Comment:
```suggestion
  session_name = "mudlet-package-repo-session",
```
This is needed, otherwise the server errors and doesn't start.